docs/root/configuration/http/http_filters/health_check_filter.rst

```diff
@@ -16,3 +16,21 @@ Health check
   <operations_admin_interface_healthcheck_fail>` admin endpoint has been called. (The
   :ref:`/healthcheck/ok <operations_admin_interface_healthcheck_ok>` admin endpoint reverses this
   behavior).
+
+Statistics
+----------
+
+The health check filter outputs statistics in the ``<stat_prefix>.health_check.`` namespace.
```

Comment:
Clarified it. Please let me know if you think we must do this part - _"adding stats support as optional (i.e., only if the stat_prefix is set). This adds the ability to avoid paying the stats costs if they are not needed."_